src/ServiceInsight/ServiceControl/DefaultServiceControl.cs

```diff
@@ -537,6 +557,7 @@ void LogError(IRestResponse response)
         static bool HasSucceeded(IRestResponse response) => successCodes.Any(x => response != null && x == response.StatusCode && response.ErrorException == null);
 
         static IEnumerable<HttpStatusCode> successCodes = new[] { HttpStatusCode.OK, HttpStatusCode.Accepted, HttpStatusCode.NotModified, HttpStatusCode.NoContent };
+        static Regex linkExpression = new Regex(@"<([^>]+)>;\s?rel=""(\w+)"",?\s?", RegexOptions.Compiled);
```

Comment:
yeah, sure. The first part `<([^>]+)>;` matches the url enclosed in `<` and `>` chars and a semicolon. Then there is an optional space and the rel part (`rel=""(\w+)""`). In the end there is an optional comma and space which separate one link from the other.

At first I tried to split on the commas to separate links and then on the semicolon to separate URL from rel part but commas can be also present in the URL. I would have to split only on the commas which are not between `<` and `>` which cannot be done declaratively but requires me to go through the string, char by char, counting `<` and `>`. 

I don't have a strong opinion and I am happy to rewrite it iteratively is you prefer this way.